doc/chrome-webstore-permissions-descriptions.md

```diff
@@ -0,0 +1,29 @@
+# Privacy tab in chrome web store:
+-----------------------------------
+
+## Permissions:
+
+#### Privacy -
+Privacy Badger needs access to the privacy API so that it can restrict the webRTCIPHandlingPolicy so that none of the user's local address information is exposed, as well as if they're in an incognito window, it will override and disable the alternateErrorPagesEnabled and hyperlinkAuditingEnabled settings.
+
+#### Cookies -
+Privacy Badger needs access to the cookies API in order to correct when Cloudflare domains are incorrectly identified as trackers, as well as to check for availability of the `firstPartyDomain` parameter in Firefox profiles.
+
+#### Storage -
+Privacy Badger needs access to the storage API so that the extension's storage and user's settings will persist beyond a browser session.
+
+#### WebRequest -
+Privacy Badger needs access to the WebRequest API so that it can view and intercept network requests being made in the browser. If user tracking is taking place via network request headers, Privacy Badger needs to be able to intercept and modify those headers. It views and then asynchronously logs the destination for outgoing requests that's flagged as tracking the user.
```

Comment:
Modification should be covered by our answer to WebRequestBlocking.

Reply:
Basically, WebRequest (non-blocking/async `webRequest` listeners in heuristicblockingaccounting.js) is for detecting tracking from headers.

WebRequestBlocking (blocking/sync `webRequest` listeners in webrequest.js) is for blocking requests and modifying headers.